packages/ui/src/views/repo/pull-request/components/pull-request-header.tsx

```diff
@@ -84,7 +84,7 @@ export const PullRequestHeader: React.FC<PullRequestTitleProps> = ({
           </Button>
           <Separator orientation="vertical" className="mx-1 h-4 bg-cn-background-0" />
           <Button variant="link" onClick={() => setIsEditing(true)}>
-            Add a description
+            {description ? 'Edit description' : 'Add a description'}
```

Comment:
Should we be localizing this text?

Reply:
Added temporarily, this section will be removed soon. It is not as per the design.